docs/releases/1.8.5.txt

```diff
@@ -43,3 +43,6 @@ Bugfixes
 * Moved the :ref:`unsaved model instance assignment data loss check
   <unsaved-model-instance-check-18>` on reverse relations to ``Model.save()``
   (:ticket:`25160`).
+
+* Re-added inline foreign keys to form instances when validating model formsets
```

Comment:
No dash is needed in "Re-added" -- does the spelling checker complain? Otherwise, good.